README.md

```diff
@@ -48,6 +48,15 @@ Occasional UI bugs may occur – in this case, please create an issue.
 - To save JWT token, [Flutter Secure Storage](https://pub.dev/packages/flutter_secure_storage) is used
 - If you find something in code that looks a bit odd, it might be some useful extension method from `lib/extensions`
 
+#### Setup instructions
```

Comment:
Could you have this list as a numbered list? Makes steps more clear.
1. aaaaa
2. bbbb
3. cccc

Reply:
aaa... ok..although this is not part of my setup instructions part, but I'll change it as per your request. 😉

Reply:
@isabelcosta, do you want the tile to be numbered, right? on setup instructions only? how about the rest of the README? should I do the same to the other parts? how bout consistency across the docs?